modules/update-conditional-web-console.adoc

```diff
@@ -15,7 +15,7 @@ You can view and assess the risks associated with particular updates with condit
 
 * Pause all `MachineHealthCheck` resources.
 
-* Your Operators that were previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update.
+* You have updated all Operators previously installed through Operator Lifecycle Manager (OLM) to a version that is compatible with your target release. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update. See "Updating installed Operators" in the additional resources section for more information on how to check compatibility and, if necessary, update the installed Operators.
```

Comment:
The "Additional resources" section should be titled appropriately.

```adoc
...in the "Additional resources" section...
```

Reply:
I always get confused about the formatting for this, thanks for clarifying.